pygmo/docstrings.cpp

```diff
@@ -2373,6 +2373,104 @@ See also the docs of the relevant C++ method :cpp:func:`pagmo::nsga2::get_log`.
 )";
 }
 
+std::string gaco_docstring()
+{
+    return R"(__init__(gen = 100, ker = 63, q = 1.0, oracle = 0., acc = 0.01, threshold = 1u,
+n_gen_mark = 7u, impstop = 100000u, evalstop = 100000u, focus = 0.,
+paretomax = 10u, epsilon = 0.9, memory = false, seed = random)
+
+Extended Ant Colony Optimization algorithm (GACO).
+
+ACO is inspired by the natural mechanism with which real ant colonies forage food.
+This algorithm has shown promising results in many trajectory optimization problems.
+The first appearance of the algorithm happened in Dr. Marco Dorigo's thesis, in 1992.
+ACO generates future generations of ants by using the a multi-kernel gaussian distribution
+based on three parameters (i.e., pheromone values) which are computed depending on the quality
+of each previous solution. The solutions are ranked through an oracle penalty method.
+
+
+The version implemented in pagmo can be applied to box-bounded multiple-objective optimization and its implementation is an extension of Schlueter's originally proposed ACO algorithm.
```

Comment:
Schlueter algorithm is called extended ACO. you may want to summarize briefly here why is our implementation differing from his.